packages/cloud-config/build.yaml

```diff
@@ -6,12 +6,6 @@ steps:
 - mkdir -p /system /oem
 - cp -rfv oem /system
 - chmod -R 600 /system/oem
-{{ if .Values.distribution }}
-{{if eq .Values.distribution "opensuse" }}
```

Comment:
We moved urls from https://quay.io/repository/costoolkit/releases-opensuse to https://quay.io/repository/costoolkit/releases-green, this was an oversight because the URL were generated by the "distribution" field which was untouched